examples/README.md

```diff
@@ -12,4 +12,7 @@ A web demo based on [Gradio](https://www.gradio.app/) is provided in repo.
 Support list:
 - ChatGLM
 - ChatGLM2
-- Llama2-chat
\ No newline at end of file
+- ChatGLM3
+- Llama2-chat
```

Comment:
Llama2 ？

Reply:
Only for chat fine-tune version since llama2 has base and chat version and chatGLM just has one version. Llama1 doesn't have chat version.